<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropForm</class>
 <widget class="QWidget" name="PropForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeWidget" name="treeProp">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">2</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>71</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Value:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtPropName">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtPropValue">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="cmdApply">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>40</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
